cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,22 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const { email, username, password } = generateUser();
+
+    cy.visit(Cypress.config().baseUrl);
+
+    cy.get('[placeholder=Username]')
+      .type(username);
+
+    cy.get('[placeholder=Email]')
+      .type(email);
+
+    cy.get('[placeholder=Password]')
+      .type(password);
+
+    cy.get('button[type=submit]').click();
 
+    cy.get('a.nav-link').contains(username).should('exist');
```

Comment:
It's good that you are checking if the username appears on the navigation bar after registration, but you should also add some assertions to check if the registration was successful. For example, you could check if a success message appears on the screen or if the user is redirected to the correct page. This will make your tests more robust and reliable.